### PR TITLE
update all vue.js dependencies to latest

### PR DESCRIPTION
**Pull request by phoet**
there are mayor version changes for vue-router and vuex, but since those are breaking changes for TypeScript only, i assume that it's safe to integrate them. the TS compiler should catch the incompatibilities.

### CODE REVIEW

**Comment by phoet** [comment]
oh, i did not actually see #45

**Comment by YurySolovyov** [comment]
Can you mention the thing about possible TS breakage in the Readme?

File: Readme.md [hunk 1]
```diff
@@ -9,10 +9,15 @@ Rails 3.1+ asset-pipeline gem to provide Vue.js
...
-- vuex v2.3.0
+- vue v2.5.13
+- vue-router v3.0.1
+- vue-resource v1.3.5
+- vuex v3.0.1
+
+### Compatibility
+
+- vue-router v3.0.0 has breaking changes for TypeScript: https://github.com/vuejs/vue-router/releases/tag/v3.0.0
+- vuex v3.0.0 has breaking changes for TypeScript: https://github.com/vuejs/vuex/releases/tag/v3.0.0
```
Comment (phoet) [inline Readme.md]: added a note about compatibilty

Comment (YurySolovyov) [reply Readme.md]: Nice!

**Comment by phoet** [comment]
it would be great to merge one or the other ASAP so that another gem version can be released. unfortunately, i can't use this through a github-gemfile thingie.

**Comment by phoet** [comment]
is it possible to move any of these branches forward and create a new gem release? otherwise i need to push a forked version to rubygems within the next days.

**Comment by YurySolovyov** [comment]
@phoet you've used full and minified UMD builds, right?
https://vuejs.org/v2/guide/installation.html#Terms

**Comment by phoet** [comment]
yes, i extracted the release zip and took the .js and .min.js of each project respectively.

**Review by adambutler — APPROVED** [review]

**Comment by phoet** [comment]
this PR has been approved for 2 weeks now. it would be great to merge it and push a new release to rubygems.

**Comment by YurySolovyov** [comment]
I'd be glad to, but I don't have merge rights. @adambutler @kimyu92 ?

**Comment by adambutler** [comment]
👋 Sorry about that... I would have thought by approving a PR that would have given you the ability to merge.

🚀 I've merged this and pushed `v2.5.13` gem to rubygems.org

**Pull request merged**